cypress-custom/integration/fee.test.ts

```diff
@@ -21,42 +22,47 @@ describe('Fetch and persist fee', () => {
   beforeEach(() => {
     cy.visit('/swap')
 
-    // Alias localStorage
     cy.window()
       .then(window => window.localStorage)
       .as('localStorage')
   })
 
   it('Persisted when selecting a token', () => {
-    const TOKEN = 'ETH'
-    const NETWORK = ChainId.RINKEBY.toString()
-    // GIVEN: A fee for a token is not in the local storage
-    // WHEN: When the user select this token
+    const DAI = '0xc7AD46e0b8a400Bb3C915120d284AafbA8fc4735'
+    const Rinkeby = ChainId.RINKEBY.toString()
 
-    // Clear localStorage
-    cy.get<Storage>('@localStorage').then(storage => storage.clear())
+    // GIVEN: Clean local storage
+    cy.clearLocalStorage()
 
-    // THEN: The fee is persisted in the local storage (redux_localstorage_simple_fee)
-    cy.get<Storage>('@localStorage')
-      .its('redux_localstorage_simple_fee')
-      .should($feeStorage => {
-        // we need to parse JSON
-        const feeStorage = JSON.parse($feeStorage)
-        const feeQuote = feeStorage[NETWORK][TOKEN]
-        expect(feeStorage).to.have.property(NETWORK)
-        expect(feeStorage[NETWORK]).to.have.property(TOKEN)
-        expect(feeQuote.token).to.equal(TOKEN)
-        expect(feeQuote.fee).to.have.property('minimalFee')
-        expect(feeQuote.fee).to.have.property('feeRatio')
-        expect(feeQuote.fee).to.have.property('expirationDate')
+    // WHEN: Select DAI token
+    cy.swapSelectInput(DAI)
+
+    // THEN: The fee for DAI is fetched
+    cy.window()
+      .then(window => window.localStorage)
+      .its(FEE_QUOTES_LOCAL_STORAGE_KEY)
+      .should(feeQuotesStorage => {
+        // THEN: The fee information in the local storage
+        if (!feeQuotesStorage) assert.fail('No fee in local storage')
```

Comment:
is this required with `.its`? shouldn't be

Reply:
since `its` is checking the key exists

Reply:
you right, this was leftovers from befor I had `should`. Good catch